features/contract-selector/ContractSelector.tsx

```diff
@@ -0,0 +1,108 @@
+import styled from "styled-components";
```

Comment:
This is an exact copy of EmpSelector with just a couple elements changed

Reply:
What changed? I wonder if we can refactor the shared logic into a FinancialContractSelector file